### PR TITLE
[feature][doc][config]Add missing transaction config in standalone.conf and broker.conf

### PR DESCRIPTION
**Pull request by liangyepianzhou**
Add doc for https://github.com/apache/pulsar/pull/15137
### Motivation & Modification
Add missing transaction config in standalone.conf and broker.conf
### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `no-need-doc` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-added`
(Docs have been already added)

### CODE REVIEW

File: conf/broker.conf [hunk 1]
```diff
@@ -1396,7 +1396,7 @@ transactionCoordinatorEnabled=false
 transactionMetadataStoreProviderClassName=org.apache.pulsar.transaction.coordinator.impl.MLTransactionMetadataStoreProvider
 
 # Transaction buffer take snapshot transaction count
-transactionBufferSnapshotMaxTransactionCount=1000
+transactionBufferSnapshotMinTransactionCount=1000
```
Comment (congbobo184) [inline conf/broker.conf:1399]: why change this to min transaction count?

File: site2/docs/reference-configuration.md [hunk 2]
```diff
@@ -337,9 +337,13 @@ brokerServiceCompactionThresholdInBytes| If the estimated backlog size is greate
...
+|transactionBufferClientOperationTimeoutInMills| The transaction buffer client's operation timeout in milliseconds.|3000|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |3000|
```
Comment (congbobo184) [inline site2/docs/reference-configuration.md:340]: two 3000

File: site2/docs/reference-configuration.md [hunk 3]
```diff
@@ -337,9 +337,13 @@ brokerServiceCompactionThresholdInBytes| If the estimated backlog size is greate
...
+|transactionCoordinatorEnabled| Whether to enable transaction coordinator in broker.|false|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |true|
```
Comment (congbobo184) [inline site2/docs/reference-configuration.md:341]: wht false and true

File: site2/docs/reference-configuration.md [hunk 4]
```diff
@@ -708,7 +712,11 @@ You can set the log level and configuration in the  [log4j2.yaml](https://github
 |replicationConnectionsPerBroker|   |16|
 |replicationProducerQueueSize|    |1000|
 | replicationPolicyCheckDurationSeconds | Duration to check replication policy to avoid replicator inconsistency due to missing ZooKeeper watch. When the value is set to 0, disable checking replication policy. | 600 |
+|transactionCoordinatorEnabled| Whether to enable transaction coordinator in broker.|false|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |true|
```
Comment (congbobo184) [inline site2/docs/reference-configuration.md:715]: true and false

**Review by codelipenghui — APPROVED** [review]

File: conf/broker.conf [hunk 5]
```diff
@@ -1395,7 +1395,7 @@ fileSystemURI=
 transactionCoordinatorEnabled=false
 transactionMetadataStoreProviderClassName=org.apache.pulsar.transaction.coordinator.impl.MLTransactionMetadataStoreProvider
 
-# Transaction buffer take snapshot transaction count
+# Transaction Buffer will take a snapshot after the number of transaction operations reaches this value.
```
Comment (Anonymitaet) [inline conf/broker.conf:1398]:
```suggestion
# Transaction buffer takes a snapshot after the number of transaction operations reaches this value.
```

1. Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

2. Why capitalize B for buffer?

File: site2/docs/reference-configuration.md [hunk 6]
```diff
@@ -338,8 +338,12 @@ brokerServiceCompactionThresholdInBytes| If the estimated backlog size is greate
...
+|transactionCoordinatorEnabled| Whether to enable transaction coordinator in broker.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |false|
+|transactionMetadataStoreProviderClassName||org.apache.pulsar.transaction.coordinator.impl.MLTransactionMetadataStoreProvider|
```
Comment (Anonymitaet) [inline site2/docs/reference-configuration.md:342]: The description is missing

File: site2/docs/reference-configuration.md [hunk 7]
```diff
@@ -338,8 +338,12 @@ brokerServiceCompactionThresholdInBytes| If the estimated backlog size is greate
...
+|transactionBufferSnapshotMaxTransactionCount|Transaction Buffer will take a snapshot after the number of transaction operations reaches this value.|1000|
+|transactionBufferSnapshotMinTimeInMillis|Transaction buffer take snapshot interval time (Unit : millisecond)|5000|
```
Comment (Anonymitaet) [inline site2/docs/reference-configuration.md:344]:
```suggestion
|transactionBufferSnapshotMinTimeInMillis|The interval time of transaction buffer takes a snapshot (in milliseconds)|5000|
```

File: site2/docs/reference-configuration.md [hunk 8]
```diff
@@ -338,8 +338,12 @@ brokerServiceCompactionThresholdInBytes| If the estimated backlog size is greate
...
+|transactionPendingAckLogIndexMinLag|MLPendingAckStore maintains a ConcurrentSkipListMap pendingAckLogIndex. It stores the position in pendingAckStore as its value and saves a position used to determine whether the previous data can be cleaned up as a key. transactionPendingAckLogIndexMinLag is used to configure the minimum lag between indexes|500|
```
Comment (Anonymitaet) [inline site2/docs/reference-configuration.md:345]: who `stores the position in pendingAckStore as its value and saves a position ....`? can you specify it explicitly?

**Comment by momo-jun** [comment]
@liangyepianzhou this PR adds not only missing configs but also new configs to align with #15137, correct? 
If yes, can you please follow the PR naming convention to add [feature] ahead of [doc] to show the dependency with #15137? This will help everyone to better understand your PR.

File: site2/docs/reference-configuration.md [hunk 9]
```diff
@@ -338,8 +338,12 @@ brokerServiceCompactionThresholdInBytes| If the estimated backlog size is greate
...
+|transactionMetadataStoreProviderClassName|The class name of transactionMetadataStoreProvider.|org.apache.pulsar.transaction.coordinator.impl.MLTransactionMetadataStoreProvider|
+|transactionBufferSnapshotMaxTransactionCount|Transaction buffer takes a snapshot after the number of transaction operations reaches this value.|1000|
+|transactionBufferSnapshotMinTimeInMillis|The interval time of transaction buffer takes a snapshot (in milliseconds).|5000|
```
Comment (momo-jun) [inline site2/docs/reference-configuration.md:344]:
```suggestion
|transactionBufferSnapshotMinTimeInMillis|The interval between two snapshots that the transaction buffer takes (in milliseconds).|5000|
```

File: site2/docs/reference-configuration.md [hunk 10]
```diff
@@ -338,8 +338,12 @@ brokerServiceCompactionThresholdInBytes| If the estimated backlog size is greate
...
+|transactionPendingAckLogIndexMinLag|MLPendingAckStore maintains a ConcurrentSkipListMap pendingAckLogIndex. PendingAckLogIndex stores the position in pendingAckStore as its value and saves a position used to determine whether the previous data can be cleaned up as a key. transactionPendingAckLogIndexMinLag is used to configure the minimum lag between indexes (keys).|500|
```
Comment (momo-jun) [inline site2/docs/reference-configuration.md:345]:
```suggestion
|transactionPendingAckLogIndexMinLag| Used to configure the minimum lag between `PendingAckLogIndex` (keys). `PendingAckLogIndex` stores the position in `MLPendingAckStore` as its value and saves a position used to determine whether the previous data can be cleaned up as a key.|500|
```

What's the measurement for the lag? milliseconds?

**Review by momo-jun — APPROVED** [review]
LGTM.

**Review by congbobo184 — APPROVED** [review]

**Pull request merged**